pyiron_atomistics/atomistics/structure/factory.py

```diff
@@ -407,3 +416,68 @@ def from_pymatgen(pymatgen_obj):
     @wraps(ovito_to_pyiron)
     def from_ovito(ovito_obj):
         return ovito_to_pyiron(ovito_obj)
+
+    def high_index_surface(self, element='Ni', crystal_structure='fcc', lattice_constant=3.526,
+                           terrace_orientation=None, step_orientation=None, kink_orientation=None,
```

Comment:
It looks to me like `terrace/step/kink/_orientation` are actually required input. Please either make them args (instead of kwargs), or in the body set a default like `terrace_orientation = terrace_orientation if terrace_orientation is not None else [1, 0, 0]`

(but kudos for not putting mutable datatypes like [1, 0, 0] directly as default values! That is indeed a bad idea in python)